osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -151,14 +155,14 @@ func addPackageInstallRemovalTest(pkgTestSetup []*packageManagementTestSetup) []
 		switch pkgManager {
 		case "apt":
 			instaneName = fmt.Sprintf("%s-%s", filepath.Base(debianImage), testName)
-			pkg := osconfigserver.BuildPackage("cowsay")
+			pkg := osconfigserver.BuildPackage(packageName)
 			installPkg := []*osconfigpb.Package{pkg}
-			pkg = osconfigserver.BuildPackage("cowsay")
+			pkg = osconfigserver.BuildPackage(packageName)
 			removePkg := []*osconfigpb.Package{pkg}
 			oc = osconfigserver.BuildOsConfig(testName, desc, osconfigserver.BuildAptPackageConfig(installPkg, removePkg, nil), nil, nil, nil, nil)
 			assign = osconfigserver.BuildAssignment(testName, desc, osconfigserver.BuildInstanceFilterExpression(instaneName), []string{fmt.Sprintf("projects/%s/osConfigs/%s", testProjectID, oc.Name)})
-			ss = fmt.Sprintf("%s\nwhile true; do /usr/bin/dpkg-query -s wget | sudo tee /dev/ttyS0; sleep 5; done", utils.InstallOSConfigDeb)
-			vs = fmt.Sprintf("package 'cowsay' is not installed")
+			ss = fmt.Sprintf("%s\nsudo apt-get -y install %s\nwhile true; do /usr/bin/dpkg-query -s %s | sudo tee /dev/ttyS0; sleep 5; done", utils.InstallOSConfigDeb, packageName, packageName)
```

Comment:
minor: these longish scripts might be clearer if you used a multi-line string literal. Something like

```golang
ss = `%s
sudo apt-get -y install %s
while true; do
/usr/bin/dpkg-query -s %s | sudo tee /dev/ttyS0;
sleep 5;
done`
ss = fmt.Sprintf(ss, ...)
```

Reply:
sure.. .will change them before merging